autoarray/operators/over_sampling/over_sample_util.py

```diff
@@ -407,18 +407,32 @@ def grid_2d_slim_over_sampled_via_mask_from(
                     for x1 in range(sub):
                         if use_jax:
                             # while this makes it run, it is very, very slow
-                            grid_slim = grid_slim.at[sub_index, 0].set(-(
-                                y_scaled - y_sub_half + y1 * y_sub_step + (y_sub_step / 2.0)
-                            ))
+                            grid_slim = grid_slim.at[sub_index, 0].set(
+                                -(
+                                    y_scaled
+                                    - y_sub_half
+                                    + y1 * y_sub_step
+                                    + (y_sub_step / 2.0)
+                                )
+                            )
                             grid_slim = grid_slim.at[sub_index, 1].set(
-                                x_scaled - x_sub_half + x1 * x_sub_step + (x_sub_step / 2.0)
+                                x_scaled
+                                - x_sub_half
+                                + x1 * x_sub_step
+                                + (x_sub_step / 2.0)
                             )

```

Comment:
I guess we could calculate these values outside of the if-else and just use the if-else to determine how the array is updated?

Reply:
Sounds plausible, albeit I think not touching this code until we're actively working on over sampling is a good idea.